applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -158,73 +138,73 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
+                long unsigned int start_element = element.GetProperties()[PIPE_START_ELEMENT];
 
                 KRATOS_INFO_IF("PipingLoop", this->GetEchoLevel() > 0 && rank == 0)
-                    << element.Id() << " " << startElement << std::endl;
+                    << element.Id() << " " << start_element << std::endl;
 
-                if (element.Id() == startElement) {
-                    PipeElementStartX = element.GetGeometry().GetPoint(0)[0];
+                if (element.Id() == start_element) {
+                    pipe_element_start_x = element.GetGeometry().GetPoint(0)[0];
                 }
             }
         }
 
-        if (PipeElements.empty()) {
-            return PipeElements;
+        if (pipe_elements.empty()) {
+            return pipe_elements;
         }
 
         // Get Maximum X Value in Pipe
-        auto rightPipe = std::max_element(PipeElements.begin(), PipeElements.end(),
-                                          [](const Element* a, const Element* b) {
+        auto right_pipe = std::max_element(pipe_elements.begin(), pipe_elements.end(),
+                                           [](const Element* a, const Element* b) {
             return a->GetGeometry().GetPoint(0)[0] < b->GetGeometry().GetPoint(0)[0];
         });
 
         // Get Minimum X Value in Pipe
-        auto leftPipe = std::min_element(PipeElements.begin(), PipeElements.end(),
-                                         [](const Element* a, const Element* b) {
+        auto left_pipe = std::min_element(pipe_elements.begin(), pipe_elements.end(),
+                                          [](const Element* a, const Element* b) {
             return a->GetGeometry().GetPoint(0)[0] < b->GetGeometry().GetPoint(0)[0];
         });
 
-        double minX = leftPipe[0]->GetGeometry().GetPoint(0)[0];
-        double maxX = rightPipe[0]->GetGeometry().GetPoint(0)[0];
+        double min_x = left_pipe[0]->GetGeometry().GetPoint(0)[0];
+        double max_x = right_pipe[0]->GetGeometry().GetPoint(0)[0];
 
         KRATOS_INFO_IF("PipingLoop", this->GetEchoLevel() > 0 && rank == 0)
-            << minX << " " << maxX << " " << PipeElementStartX << std::endl;
+            << min_x << " " << max_x << " " << pipe_element_start_x << std::endl;
 
-        if ((minX != PipeElementStartX) && (maxX != PipeElementStartX)) {
+        if ((min_x != pipe_element_start_x) && (max_x != pipe_element_start_x)) {
             KRATOS_ERROR << "Unable to determine pipe direction (multiple directions possible) -  "
                             "Check PIPE_START_ELEMENT"
                          << std::endl;
         }
 
-        if (minX == PipeElementStartX) {
+        if (min_x == pipe_element_start_x) {
             // Pipe Left -> Right
-            sort(PipeElements.begin(), PipeElements.end(), [](const Element* lhs, const Element* rhs) {
+            sort(pipe_elements.begin(), pipe_elements.end(), [](const Element* lhs, const Element* rhs) {
```

Comment:
I would suggest to use `std::sort`.
```suggestion
            std::sort(pipe_elements.begin(), pipe_elements.end(), [](const Element* lhs, const Element* rhs) {
```

Reply:
done